### ISSUE TITLE
Get 'nan' for input after applying a convolutional layer

### ISSUE THREAD

**Issue by YilinLiu97** [pytorch/pytorch#2915]
Hi, I want to know what may be the reasons for getting 'nan' after applying a convolutional layer on the input? Thanks in advance!!

Here is part of the code:

       self.encoder_1 = nn.Sequential(
            nn.Conv3d(1,25,7,padding=6,dilation=2),
            nn.BatchNorm3d(25),
            nn.PReLU()
            ) #first encoder

Output:
encoder_1  Variable containing:
(0 ,0 ,0 ,.,.) = 
 nan nan nan nan nan nan
 nan nan nan nan nan nan
 nan nan nan nan nan nan
 nan nan nan nan nan nan
 nan nan nan nan nan nan
 nan nan nan nan nan nan

(0 ,0 ,1 ,.,.) = 
 nan nan nan nan nan nan
 nan nan nan nan nan nan
 nan nan nan nan nan nan
 nan nan nan nan nan nan
 nan nan nan nan nan nan
 nan nan nan nan nan nan

(0 ,0 ,2 ,.,.) = 
 nan nan nan nan nan nan
 nan nan nan nan nan nan
 nan nan nan nan nan nan
 nan nan nan nan nan nan
 nan nan nan nan nan nan
 nan nan nan nan nan nan

(0 ,0 ,3 ,.,.) = 
 nan nan nan nan nan nan
 nan nan nan nan nan nan
 nan nan nan nan nan nan
 nan nan nan nan nan nan
 nan nan nan nan nan nan
 nan nan nan nan nan nan

**Comment by fmassa** [comment]
Questions are better suited for the forums https://discuss.pytorch.org/
For you particular case,  make sure that your input is initialized. `torch.Tensor(2,2)` produces initialized tensors.
